src/org/labkey/test/components/html/BootstrapMenu.java

```diff
@@ -165,9 +155,9 @@ protected WebElement findVisibleMenuItemOrNull(String text)
 
     static public class Locators
     {
-        public static Locator.XPathLocator toggleAnchor()
+        public static Locator.XPathLocator dropdownToggle()
         {
-            return Locator.tagWithAttribute("*", "data-toggle", "dropdown");
+            return Locator.byClass("dropdown-toggle");
         }
```

Comment:
Why not make the locator  a public static member variable and not a function? Just curious.

Reply:
No particular reason. Just trying to minimize the changes but I suppose I'm already changing the method name so it wouldn't be much more of a change to make it a contant.